aws/codefresh-onprem/main.tf

```diff
@@ -43,15 +43,40 @@ data "terraform_remote_state" "backing_services" {
   }
 }
 
+variable "acm_enabled" {
+  description = "Set to false to prevent the acm module from creating any resources"
+  default     = "true"
+}
+
+variable "acm_primary_domain" {
+  description = "A domain name for which the certificate should be issued"
+}
+
+variable "acm_san_domains" {
+  type        = "list"
+  default     = []
+  description = "A list of domains that should be SANs in the issued certificate"
+}
+
+variable "acm_zone_name" {
+  type        = "string"
+  default     = ""
+  description = "The name of the desired Route53 Hosted Zone"
+}
+
 module "codefresh_enterprise_backing_services" {
-  source          = "git::https://github.com/cloudposse/terraform-aws-codefresh-backing-services.git?ref=tags/0.2.0"
+  source          = "git::https://github.com/cloudposse/terraform-aws-codefresh-backing-services.git?ref=added-acm"
```

Comment:
Use a released version of terraform-aws-codefresh-backing-services

Reply:
Done